docs/source/indexes/vamana.rst

```diff
@@ -0,0 +1,76 @@
+--------------------------
+
+The 'vamana::serialize' API calls writes the index to a file with a format that is compatible with the `open-source DiskANN repositoriy <https://github.com/microsoft/DiskANN>`. This allows cuVS to be used to accelerate index construction while leveraging the efficient CPU-based search currently available.
+
+Configuration parameters
+------------------------
+
+Build parameters
+~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Name
+     - Default
+     - Description
+   * - graph_degree
+     - 32
+     - The maximum degre of the final Vamana graph. The internal representation of the graph includes this many edges for every node, but serialize will compress the graph into a 'CSR' format with, potentially, fewer edges.
+   * - visited_size
+     - 64
+     - Maximum number of visited nodes saved during each traversal to insert a new node. This corresponds to the 'L' parameter in the paper.
+   * - vamana_iters
+     - 1
+     - Number of iterations ran to improve the graph. Each iteration involves inserting every vector in the dataset.
+   * - alpha
+     - 1.2
+     - Alpha parameter that defines how aggressively to prune edges.
+   * - max_fraction
+     - 0.06
+     - Maximum fraction of the dataset that will be inserted as a single batch. Larger max batch size decreases graph quality but improves speed.
+   * - batch_base
+     - 2
+     - Base of growth rate of batch sizes. Insertion batch sizes increase exponentially based on this parameter until max_fraction is reached.
+   * - queue_size
+     - 127
+     - Size of the candidate queue structure used during graph traversal. Must be (2^x)-1 for some x, and must be > visited_size.
+
+Tuning Considerations
+---------------------
+
+The 2 hyper-parameters that are most often tuned are `graph_degree` and `visited_size`. The time needed to create a graph increases dramatically when increasing `graph_degree`, in particular. However, larger graphs may be needed to achieve very high recall search, especially for large datasets.
+
+Memory footprint
+----------------
+
+Vamana builds a graph that is stored in device memory. However, in order to serialize the index and write it to a file for later use, it must be moved into host memory. If the `include_dataset` parameter is also set, then the dataset must be resident in host memory when calling serialize as well. 
+
+Device memory usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The built index represents the graph as fixed degree, storing a total of :math:`graph_degree * n_index_vectors` edges. Graph construction also requires the dataset be in device memory (or it copies it to device during build). In addition, device memory is used during construction to sort and create the reverse edges. Thus, the amount of device memory needed depends on the dataset itself, but it is bounded by a maximum sum of:
+
+- vector dataset: :math:`n_index_vectors * n__dims * sizeof(T)`
```

Comment:
I think you might need to escape all of the _'s in here. Have you checked to see how this formats?

Reply:
Did a quick scan but didn't look at all the formatting details. Thanks for checking, I'll do a pass over the format of the final built docs.

Reply:
Fixed.